datasets/batch_view/longitudinal/intro.md

```diff
@@ -24,6 +24,8 @@ In summary, the longitudinal table differs from `main_summary` in two important
 * The longitudinal dataset groups all data so that one row represents a client_id
 * The longitudinal dataset samples to 1% of all client_ids
 
+Please note that this dataset only contains pre-release (or opt-in) histograms and scalars.
```

Comment:
Isn't the opposite true? We [only include opt-out histograms](https://github.com/mozilla/telemetry-batch-view/blob/master/src/main/scala/com/mozilla/telemetry/views/Longitudinal.scala#L111) in longitudinal?

Reply:
Of course you're right :)